src/Deferred.php

```diff
@@ -122,6 +130,9 @@ public function reject(ClientExceptionInterface $exception): void
         }
     }
 
+    /**
+     * {@inheritDoc}
+    */
```

Comment:
we don't do inheritdoc when there is nothing else. if there is no docblock, most tools take the docblock of the parent/interface